patches/api/0299-Missing-Entity-API.patch

```diff
@@ -76,6 +76,58 @@ index 0000000000000000000000000000000000000000..39ad7d283609d7e427a2ab35b6fad839
 +    SchoolableFish getSchoolLeader();
 +
 +}
+diff --git a/src/main/java/io/papermc/paper/potion/PotionEffectPair.java b/src/main/java/io/papermc/paper/potion/PotionEffectPair.java
+new file mode 100644
+index 0000000000000000000000000000000000000000..60c44aaf3339ab7fc66a342de2a499a5c0917189
+--- /dev/null
++++ b/src/main/java/io/papermc/paper/potion/PotionEffectPair.java
+@@ -0,0 +1,46 @@
++package io.papermc.paper.potion;
++
++import org.bukkit.potion.PotionEffectType;
++import org.jetbrains.annotations.NotNull;
++
++/**
++ * Represents an effect paired with a duration.
++ */
++public interface PotionEffectPair {
```

Comment:
This should get a less generic name, it sounds like it _holds_ one or a pair of potion effects (not a type), when really it is just a bare bones potion effect. 

But I also don't know what 😎